sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 1.0.0-preview.4 (Unreleased)
+## 1.0.0-preview.4 (2020-06-08)
+
+- [Breaking] Refactored `SearchServiceClient` and split it to `SearchIndexClient` and `SearchIndexerClient`.[#8986](https://github.com/Azure/azure-sdk-for-js/issues/8986)
+- [Breaking] Changed `SearchIndexClient` class to `SearchClient` class.[#9014](https://github.com/Azure/azure-sdk-for-js/issues/9014)
```

Comment:
can we merge this into the previous bullet? It seems confusing to say you split something into two names and then renamed one on the very next line

Reply:
DOne